konf/site.yaml

```diff
@@ -36,6 +36,16 @@ env:
   - name: SENTRY_DSN
     value: https://0293bb7fc3104e56bafd2422e155790c@sentry.is.canonical.com//13
 
+  - name: DISCOURSE_API_KEY
+    secretKeyRef:
+      key: api_key
+      name: discourse-api
```

Comment:
The secret is named `discourse-api` for demos, but for production and staging you went with just `discourse`.. It needs to be the same everywhere and updated here if you choose to go with `discourse`.